CTF/the_simpsons.md

```diff
@@ -0,0 +1,24 @@
+Challenge link: https://ctflearn.com/challenge/160
+
+
+Challenge category: Cryptography, hard 
+
+Challenge points: 80 points
+
+Challenge description:
+
+Ya know, I was thinking... wouldn't the Simpsons use octal as a base system? They have 8 fingers... Oh, right! The problem! Ummm, something seems odd about this image... https://mega.nz/#!yfp1nYrQ!LOz_eucuKkjAaDqVvz3GWgbfdKWn8BhussKZbx6bUMg
+
+Challenge solution:
+
+1. firstly, go to the link provided and download it as jpg.
+2. open the jpg with nodepad -- in case there is sth hidden 
+3. at the button there are lines of python code 
```

Comment:
Could you elaborate? if I am a new reader (who did not go through the session), I would not have understood what you meant